DRAFT.md

```diff
@@ -734,7 +734,7 @@ program is ill-formed. Every object of type `indirect<T, Allocator>` uses an
 object of type `Allocator` to allocate and free storage for the owned object as
 needed.
 
-3. Constructing an owned object with `args...` using `a`
+3. Constructing an owned object with `args...` using the allocator `a`
```

Comment:
We got too carried away simplifying the "constructs an owned object with args... using the allocator a" wording and broke the definitions to no longer match the way they're used.  This reverts just enough to restore things